docs/en/reference/attributes-reference.rst

```diff
@@ -262,10 +265,11 @@ Examples:
     )]
     protected $loginCount;
 
-    // MySQL example: full_name char(41) GENERATED ALWAYS AS (concat(firstname,' ',lastname)),
+    // columnDefinition is raw SQL, not DQL. This example works for MySQL:
     #[Column(
         type: "string",
         name: "user_fullname",
+        columnDefinition: "VARCHAR(255) GENERATED ALWAYS AS (concat(firstname,' ',lastname))",
```

Comment:
i guess this was what was intended above. the comment was rather confusing and misleading before